src/containers/BooksForm.js

```diff
@@ -11,17 +40,19 @@ const BooksForm = () => {
     'Sci-Fi',
   ];
   return (
-    <div className="form">
-      <input placeholder="Title" />
-      <select>
+    <form className="form" id="form">
+      <input name="title" placeholder="Title" onChange={handleChange} />
+      <select name="category" onChange={handleChange}>
         {options.map((option) => (
           <option key={Math.random()} value={option}>
             {option}
           </option>
         ))}
       </select>
-      <button type="button">ADD</button>
-    </div>
+      <button type="button" onClick={handleClick}>
```

Comment:
For the booksForm handleSubmit function, It has to be attached to the form's onSubmit event. This way you can add validation to the input fields by adding the required attribute, and write less code as it also one of jsbest practices.

![image](https://user-images.githubusercontent.com/31897434/117916838-d4a58300-b2df-11eb-8575-689256559af2.png)